DXRModules/DeusEx/Classes/DXRStartMap.uc

```diff
@@ -87,11 +128,13 @@ function PreFirstEntry()
 
     case "12_VANDENBERG_CMD":
         if (dxr.flags.settings.starting_map >= 121) {
+            RemoveJock('UN_BlackHeli');
             foreach AllActors(class'#var(DeusExPrefix)Mover', dxMover, 'comhqdoor') {
                 dxMover.InterpolateTo(1, 0.0);
                 break;
             }
             dxr.flagbase.SetBool('DL_TonyScared_Played', true,, 15); // You won't find cover in the comm building.
+            AddNoteFromConv('DL_no_carla', true);
```

Comment:
why is this here instead of in `PreFirstEntryStartMapFixes` with all the other calls to AddNoteFromConv? this is a duplicate of 

`AddNoteFromConv('MeetCarlaBrown', bEmptyNotes, 0); // Backup power for the bot security system`

they both give the code 5868

we really just need to delete the `break;` on line 812 https://github.com/Die4Ever/deus-ex-randomizer/blob/3024aa778432dca3dc74a3224a009c761d70c006/DXRModules/DeusEx/Classes/DXRStartMap.uc#L812